pkg/ebpf/tracer.go

```diff
@@ -886,6 +886,30 @@ func connExistsFn(procRoot string) (func(*ConnTuple) bool, func()) {
 			return false
 		}
 
+		if ok {
+			return ok
+		}
+
+		conn = netlink.Con{
+			Con: ct.Con{
+				Origin: &ct.IPTuple{
+					Src: &dstAddr,
+					Dst: &srcAddr,
+					Proto: &ct.ProtoTuple{
+						Number:  &protoNumber,
+						SrcPort: &dstPort,
+						DstPort: &srcPort,
+					},
+				},
+			},
+		}
+
+		ok, err = ctrk.Exists(&conn)
```

Comment:
Is it possible to do a batch query to conntrack so we can query both tuples at once?

Reply:
Not that I could find.